supplementary_style_guide/style_guidelines/links.adoc

```diff
@@ -2,11 +2,44 @@
 [[links]]
 = Links
 
-// [[internal-links]]
-// == Internal links
+[[cross-references]]
+== Cross-references
 
-// [[external-links]]
-// == External links
+Follow these guidelines when adding cross-references within your documentation:
+
+* Include cross-references only when necessary.
+* If the information is critical, consider including it instead of cross-referencing.
+
+.Example AsciiDoc: Cross-reference
+----
+For more information about <topic>, see xref:<link>[<link_text>].
+----
+
+[[external-links]]
+== External links
+
+Follow these guidelines when linking externally:
+
+* Avoid links to external sites when possible. External links can change or be unreliable.
+* Use hyperlinks unless the URL is an example URL or is otherwise inaccessible to users.
```

Comment:
Consider cross-referencing the ISG guidance on sample URLs: the "Using addresses in examples" section here: https://www.ibm.com/docs/en/ibm-style?topic=elements-web-ip-email-addresses

Reply:
@bburt-rh So we don't actually ever link to the ISG from the SSG (since the SSG is public, and the ISG is not available to everyone outside of RH). We also avoid naming the sections too, but if you think it's worthwhile I could add something like 

```
See the _IBM Style Guide_ for information about using addresses in examples.
```

wdyt?

Reply:
Thanks, updated!